lib/util/Consts.ts

```diff
@@ -3,11 +3,17 @@ import type * as RDF from 'rdf-js';
 
 const DF = new DataFactory();
 
-export const TRUE_STR = '"true"^^xsd:boolean';
-export const FALSE_STR = '"false"^^xsd:boolean';
-export const EVB_ERR_STR = '"not an dateTime"^^xsd:dateTime';
+export type LiteralTypes = TypeAlias | TypeURL;
+
+export enum TypeAlias {
+  // Numeric is everything defined in https://www.w3.org/TR/sparql11-query/#operandDataTypes
+  SPARQL_NUMERIC = 'SPARQL_NUMERIC',
+  SPARQL_STRINGLY = 'SPARQL_STRINGLY',
```

Comment:
Do we have one for this? I'd need to check it. I thought you suggested creating this type for some cleaner code?

Reply:
Yes, this definitely needs some kind of description, because this doesn't appear to be part of the SPARQL/RDF spec (?).

Reply:
I did suggest it indeed, and it is present in the spec, but it is hidden a bit (can't put all the details up front ofc).

The first paragraphs of https://www.w3.org/TR/sparql11-query/#func-strings provide some more context. There is a separate section on argument compatibility that is relevant for some other issues as well. Let's summarize this together with the link and add it to the code.

Reply:
> take a string literal as an argument and accept a simple literal, a plain literal with language tag, or a literal with datatype xsd:string
This means a simple literal. Whatever that means, is also a sparql_stringly. We should make sure this is True as well.

Reply:
This true because we consider simple literal (no datatype) to be xsd:string (see also https://github.com/comunica/sparqlee/pull/103#discussion_r688454246).

Reply:
Since I stumbled upon this problem like 5 times now. Should we put it in the readme or would that be overkill?

Reply:
I'll just write it very clearly in the places I had issues with it. If I put it in the readme the chance is low someone will read it.

Reply:
I don't think it's necessary. Maybe mentioning simple literals don't exists because we make them xsd:string is worth it.